uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/leave-request-popup-controller.js

```diff
@@ -178,13 +191,19 @@ define([
        * @returns {Boolean}
        */
       vm.canSubmit = function () {
-        return vm.leaveRequest.from_date && vm.leaveRequest.to_date &&
+        var canSubmit = vm.leaveRequest.from_date && vm.leaveRequest.to_date &&
           vm.leaveRequest.to_date_type && vm.leaveRequest.from_date_type;
+
+        if (canEdit()) {
+          canSubmit = canSubmit && vm.uiOptions.selectedStatus;
```

Comment:
changed logic to 
```javascript
        //check if manager has changed status
        if (vm.uiOptions.isManager) {
          canSubmit = canSubmit && vm.uiOptions.selectedStatus;
        }
```
its not exactly edit mode, its to check if manager can update or not.